security-command-center/snippets/src/main/java/vtwo/source/GetSource.java

```diff
@@ -38,25 +38,23 @@ public static void main(String[] args) {
   }
 
   // Demonstrates how to retrieve a specific source.
-  public static Source getSource(String organizationId, String sourceId) {
+  public static Source getSource(String organizationId, String sourceId) throws IOException {
     // Initialize client that will be used to send requests. This client only needs to be created
     // once, and can be reused for multiple requests.
-    try (SecurityCenterClient client = SecurityCenterClient.create()) {
```

Comment:
Kindly add back the try() {} block in all the snippets. Don't add a catch, instead throw the exceptions

Reply:
done.